src/mina-signer/TSTypes.ts

```diff
@@ -0,0 +1,66 @@
+export type UInt32 = number | bigint | string;
```

Comment:
this file was just dumped here from Mina/mina-signer and doesn't do anything currently